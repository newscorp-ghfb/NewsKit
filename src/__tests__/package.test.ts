const {newPackageJson} = require('../../scripts/build-package-json-support');

describe('package.json updated', () => {
  test('remind developer that dependenciesNotToBeIncluded array may need updating', () => {
    const packageJson = newPackageJson();
    const {version, ...restOfPackage} = packageJson;
    expect(restOfPackage).toMatchSnapshot();
  });
});

// 'package.test.ts' cannot be compiled under '--isolatedModules' because it is considered a global script file. Add an import, export, or an empty 'export {}' statement to make it a module.
export {};
