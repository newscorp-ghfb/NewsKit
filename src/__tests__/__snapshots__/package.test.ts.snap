// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`package.json updated remind developer that dependenciesNotToBeIncluded array may need updating 1`] = `
Object {
  "dependencies": Object {
    "@capsizecss/core": "^3.0.0",
    "@emotion-icons/emotion-icon": "4.1.0",
    "@emotion-icons/material": "3.9.0",
    "@emotion-icons/material-outlined": "3.8.0",
    "@floating-ui/react-dom-interactions": "^0.6.0",
    "@newskit-themes/newskit-website": "1.0.2",
    "@newskit-themes/the-sun": "^0.1.0",
    "@seznam/compose-react-refs": "^1.0.5",
    "aria-hidden": "^1.1.3",
    "date-fns": "^2.6.0",
    "debounce": "^1.2.0",
    "dequal": "^1.0.0",
    "downshift": "^6.1.7",
    "react-focus-lock": "^2.5.0",
    "react-hook-form": "^7.43.1",
    "react-hot-toast": "^1.0.2",
    "react-range": "^1.8.12",
    "react-transition-group": "^4.4.1",
    "react-virtual": "^2.10.4",
  },
  "description": "A React Component library implementing the News design language",
  "keywords": Array [
    "react",
    "newsuk",
  ],
  "license": "BSD-3-Clause",
  "main": "cjs/index.js",
  "module": "esm/index.js",
  "name": "newskit",
  "peerDependencies": Object {
    "@emotion/react": ">= 11.1.5 <= 11.9.3",
    "@emotion/styled": ">= 11.1.5 <= 11.9.3",
    "react": ">= 16.8.0 <= 18",
    "react-dom": ">= 16.8.0 <= 18",
    "react-is": ">= 16.8.0 <= 18",
  },
  "repository": "https://github.com/newscorp-ghfb/newskit.git",
  "resolutions": Object {
    "@types/react": "18",
    "kind-of": "^6.0.3",
    "minimist": "^0.2.1",
  },
  "sideEffects": false,
  "types": "cjs/index.d.ts",
  "version": "7.3.0",
}
`;
